main.js

```diff
@@ -133,6 +151,35 @@ define(function (require, exports, module) {
         };
     }
     
+    /**
+     * Apply the whitespace colors.
+     * This does not overwrite styles already defined by a theme.
+     */
+    function _applyColors() {
+        // Compile the CSS from the template
+        var userColors = _preferences.get("colors"),
+            template   = _.template(stylesTemplate),
+            css = template({
```

Comment:
After checking the Lodash docs, I understood what you were talking about here. Changed.